addons/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/test/java/org/nuxeo/ecm/core/storage/sql/TestS3BinaryManager.java

```diff
@@ -101,16 +101,21 @@ public static void beforeClass() {
                 System.getenv(SDKGlobalConfiguration.ALTERNATE_SECRET_KEY_ENV_VAR));
         String envToken = StringUtils.defaultIfBlank(System.getenv(SDKGlobalConfiguration.AWS_SESSION_TOKEN_ENV_VAR),
                 "");
+        // Environment variable name for the AWS bucket name , prefix and region
+        // will use default variable if not provided
+        String envBucketName = StringUtils.defaultIfBlank(System.getenv("AWS_BUCKET_NAME_ENV_VAR"),"nuxeo-s3-directupload");
```

Comment:
It's strange to end the names in `_ENV_VAR`, it's redundant. I wouldn't do it.
As this is not generic AWS config but just for Nuxeo tests, it's worth adding that in their name: `AWS_BUCKET_NAME_NUXEO_TEST`
Also you should put these in a constant in a single file and reuse it from elsewhere, to avoid duplication and typos.

Reply:
Add a space after `,`

Reply:
Finally, you should do similar changes in the new `TestS3BlobStoreAbstract`.

Reply:
Hello @efge ,

thanks you , i will apply changes.